package.json

```diff
@@ -32,19 +32,23 @@
     "url": "https://github.com/CrowdStrike/eslint-config-crowdstrike/issues"
   },
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike#readme",
+  "engines": {
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
+  },
   "devDependencies": {
     "@crowdstrike/commitlint": "^7.0.0",
-    "eslint": "^8.0.0",
+    "@eslint/js": "^9.7.0",
+    "@eslint/eslintrc": "^3.1.0",
     "eslint-config-crowdstrike": "link:",
     "eslint-config-crowdstrike-node": "^3.0.0",
     "eslint-plugin-json-files": "^4.0.0",
-    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-n": "^17.9.0",
```

Comment:
Can this be done first without jumping to eslint 9?

Reply:
Yes, we would just need to bump `eslint` to `8.23.0` and set the `engine` node versions in this changeset, unless we wanted to go to `eslint-plugin-n@14.0.0`, which I think should be compatible with the existing packages

Reply:
lets do eslint-plugin-n@14.0.0 first

Reply:
https://github.com/CrowdStrike/eslint-config-crowdstrike/pull/92